ocs_ci/helpers/helpers.py

```diff
@@ -3909,3 +3909,46 @@ def get_noobaa_db_used_space():
         f"noobaa_db used space is {df_out[-4]} which is {df_out[-2]} of the total PVC size"
     )
     return df_out[-4]
+
+
+def create_storageclassclaim(
```

Comment:
Please move the function to [ocs_ci/ocs/resources/storageclassclaim.py](https://github.com/red-hat-storage/ocs-ci/pull/5935/files#diff-deff0564df2356f1bee617e0d3f5280037478ea7837236a19d4023fde20b5fac). This is directly linked to the resource module and shouldn't be in helpers.

Reply:
@fbalak Moved the function to storageclassclaim.py